### ISSUE TITLE
Error when join a managedCluster

### ISSUE THREAD

**Issue by xuezhaojun** [open-cluster-management-io/clusteradm#54]
I'm using the clusteradm v0.1.0-alpha.5v.1

After run `clusteradm join ...` command, I got this error message:

```
E0812 06:32:36.654581  454000 memcache.go:196] couldn't get resource list for operator.open-cluster-management.io/v1: the server could not find the requested resource
Error: no matches for kind "Klusterlet" in version "operator.open-cluster-management.io/v1"
```

**Comment by dongwangdw** [comment]
@xuezhaojun 
I tried with kind 0.11.0 with 1.21.1 kube version and it worked well.

You may need to provider more context about your issue.

**Comment by xuezhaojun** [comment]
It's weird that this issue doesn't appear every time. I realize my poor network environment could have an effect on this. 

At the time when the command "clusteradm join ..." was running, most pods are still in ContainerCreating phase because images are not pulled yet. 

After I changed to a good network environment, I found everything is going fine.

I  will close this issue, thanks for your time!
